test/k8s/datapath_configuration.go

```diff
@@ -810,25 +809,25 @@ func checkMonitorOutput(monitorOutput []byte, egressPktCount, ingressPktCount in
 	By("Looking for TCP notifications using the ephemeral port %q", portBytes)
 	port, err := strconv.Atoi(string(portBytes))
 	if err != nil {
-		GinkgoPrint("ephemeral port %q could not be converted to integer: %s", string(portBytes), err)
-		return false
+		return fmt.Errorf("ephemeral port %q could not be converted to integer: %s",
+			string(portBytes), err)
 	}
 
 	expEgress := fmt.Sprintf("SrcPort=%d", port)
 	expEgressRegex := regexp.MustCompile(expEgress)
 	egressMatches := expEgressRegex.FindAllIndex(monitorOutput, -1)
 	if len(egressMatches) != egressPktCount {
-		GinkgoPrint("Could not locate final FIN notification in monitor log: egressTCPMatches %+v", egressTCPMatches)
-		return false
```

Comment:
I don't understand the question. I mean these two tests are currently spamming the ginkgo logs with these GinkgoPrint messages that are huge (because of `%+v` of the regexp matches).

Reply:
Not necessarily. We have retries here, so it's possible for this to be printed and the test to then succeed in a subsequent retry.

Reply:
I'll improve the commit description before merging.